dev/NavigationView/NavigationView.xaml

```diff
@@ -576,7 +570,68 @@
                                 </VisualState>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
-                    </primitives:NavigationViewItemPresenter>
+
+
+                        <primitives:NavigationViewItemPresenter
+                            x:Name="NavigationViewItemPresenter"
+                            Icon="{TemplateBinding Icon}"
+                            ContentTransitions="{TemplateBinding ContentTransitions}"
+                            ContentTemplate="{TemplateBinding ContentTemplate}"
+                            Margin="{TemplateBinding Margin}"
+                            Padding="{TemplateBinding Padding}"
+                            Foreground="{TemplateBinding Foreground}"
+                            Background="{TemplateBinding Background}"
+                            BorderBrush="{TemplateBinding BorderBrush}"
+                            BorderThickness="{TemplateBinding BorderThickness}"
+                            UseSystemFocusVisuals="{TemplateBinding UseSystemFocusVisuals}"
+                            VerticalAlignment="{TemplateBinding VerticalAlignment}"
+                            HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                            VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                            HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                            ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}"
+                            Content="{TemplateBinding Content}"
+                            primitiveContract7Present:CornerRadius="{TemplateBinding CornerRadius}"
+                            IsTabStop="False">
+                        </primitives:NavigationViewItemPresenter>
+                        <local:ItemsRepeater
+                                Grid.Row="1"
+                                Visibility="Collapsed"
+                                x:Name="NavigationViewItemMenuItemsHost">
+                            <local:ItemsRepeater.Layout>
+                                <local:StackLayout Orientation="Vertical"/>
+                            </local:ItemsRepeater.Layout>
+                        </local:ItemsRepeater>
+                        <FlyoutBase.AttachedFlyout>
+                            <Flyout x:Name="ChildrenFlyout"
+                                    contract7NotPresent:Placement="Right"
+                                    contract7Present:Placement="RightEdgeAlignedTop">
+                                <Flyout.FlyoutPresenterStyle>
+                                    <Style TargetType="FlyoutPresenter">
+                                        <Setter Property="Padding" Value="0,8" />
```

Comment:
>0,8 [](start = 74, length = 3)

ideally these magic numbers could be exposed as static resources

Reply:
Agreed, resources would be great. I expect a small but non-zero percentage of customers wanting to customize the flyout padding. Is there an existing resource we can reuse that logically makes sense?

Reply:
That's definitely is a good idea. As of now, I can't think of an existing resource that we could re-use here, as I styled this flyout to be like the overflow flyout in TopNav, which does not expose these properties.

Reply:
@ojhad @YuliKl @StephenLPeters We could add two padding theme resources - one resource for the children flyouts (used in both Left and Top mode) and the second resource used for the overflow menu in Top mode. 

Why two resources instead of one? Logically, the content of both flyouts differ. One flyout lists the *children* for a menu item, whereas the overflow flyout lists distinct top-level items (including their children if any). As such it could make sense to give customers the ability to choose a different padding for these two different flyout types.

Proposed theme resources:
* TopNavigationViewOverflowMenuPadding
* NavigationViewItemChildrenMenuFlyoutPadding

I'm proposing using "MenuFlyout" instead of just "Menu" for the children flyout padding theme resource as children can also be displayed inline (in Left mode) and I'm not sure if `NavigationViewItemChildrenMenuPadding` would make it clear that this resource only affects the padding when the children are displayed inside a flyout and not inline. As such, I picked "MenuFlyout" to make it crystal clear when the padding will be applied.
I don't believe adding "Flyout" to the `TopNavigationViewOverflowMenuPadding` name is required as the [documentation](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#top-to-left-navigation) also only speaks of the "overflow menu" and not the "overflow menu flyout".

Thoughts?